### ISSUE TITLE
Accessibility issues found in design system audit

### ISSUE THREAD

**Issue by joeedhorton** [UKHomeOffice/design-system#168]
## What needs correcting
During an accessibility audit conducted on the design system, the following issues were found:

**Issues found relating to content** — SUE

- [x] DDaT acronym is not explained on the styles home page
- [x] Get involved is named Community in it's <title> tag

**Issues found from keyboard navigation** — SUE

- [x] Styles/Colour page — Links in the examples are active
- [x] Styles/Images — Link doesn't work in related section (guidance images)
- [x] Styles/Page template — links in the examples are active 
- [x] Components/Alerts — links in examples are active
- [x] Components/Back link — links in examples are active
- [x] Components/Buttons — links in examples are active — Related section link broken (Interruption card) 
- [x] Pattern/Interruption card button example is active
- [x] Components/Confirmation pages — links in examples are active
- [x] Components/Labels - grey box under Services using labels (no graphic)
- [x] Components/Pagination - links in examples are active
- [x] Components/Side tab - links in examples are active
- [x] Components/Sub navigation - links in example are active — link in Related section is broken (service header)
- [x] Components/Table multi-select - checkboxes are active in the example
- [x] Components/Timeline - Link in example is active
- [x] Patterns/Biometric residence card or permit details — related pattern link broken (guidance images)      
- [x] Patterns/Find information on a document — related pattern link broken (Styles/Images)
- [x] Patterns/Get more details — related pattern link broken (modal windows)  
- [x] Patterns/Sign out — related pattern link broken (service header)
- [x] On all Get in touch sections, link to Slack opens a pop up which is not active or able tab into. **Remove link**.

**Issues found from WAVE** — JOE

HOME
- [x] A heading level is skipped. Replace H4 with H3

STYLES
- [x] in Images, replace H3 with H2 ("Use of images")

COMPONENTS
- [x] Sidebar component has low text contrast with disabled nav item

PATTERNS
- [x] Change H4 in sidenav to #4E5455 to meet colour contrast stuff
- [x] Access a service, skipped headers — change H4 to H3
- [x] Enquiry form has missing form labels (issue with pattern itself)

**Comment by SueSpevack** [comment]
Accessibility issues have been addressed and pushed live
